### DISCUSSION TITLE
Wordpress ImageMagick integration "optimizing" my PNGs to 5X original size - FIX

### DISCUSSION THREAD

**Discussion by Swapot** [ImageMagick/ImageMagick#7063] [Help]

**Comment by snibgo** [comment]
I know nothing about Wordpress.

Your screenshot shows the thumbnails are PNG, so I expect that setting ‘jpeg_quality’ will make no difference.

When IM writes PNG files, the `-quality` setting doesn't affect image quality, which is always lossless. However, resizing an image will usually increase the number of colours. An input image that has 256 colours (8 bits per channel) will afer resizing have many more colours, so the PNG will probably be 16 bits per channel. It may also be saving three colour channels.

After resizing, I suggest a command line could include:
```
-colorspace Gray -colors 255
```
I don't know how to do that in Wordpress.
